peps/pep-0757.rst

```diff
@@ -148,9 +150,6 @@ Export API
    If *export_long->digits* is not ``NULL``, :c:func:`PyLong_FreeExport` must be
    called when the export is no longer needed.
 
-   On CPython 3.14, no memory copy is needed, it's just a thin wrapper to
-   expose Python int internal digits array.
```

Comment:
Why removing this sentence?

Reply:
I think that if we keep above constraints, this is more or less obvious.  Maybe it does make sense in the PEP, but not in sphinx docs for a function.

Maybe leave this as some commentary *after* function?

Reply:
I recommend leaving for now. It can always be removed later as we get closer to 3.14 if desired.